reposerver/src/test/scala/com/advancedtelematic/tuf/reposerver/util/ResourceSpec.scala

```diff
@@ -159,28 +164,22 @@ class FakeKeyserverClient(defaultKeyType: KeyType) extends KeyserverClient {
   }
 
   override def deletePrivateKey(repoId: RepoId, keyId: KeyId): Future[Unit] = FastFuture.successful {
-    keys.asScala.get(repoId).flatMap(_.values.find(_.getPublic.id == keyId)).getOrElse(throw RoleKeyNotFound)
+    keys.asScala.get(repoId).flatMap(_.values.find(_.pubkey.id == keyId)).getOrElse(throw RoleKeyNotFound)
 
-    keys.computeIfPresent(repoId, (id: RepoId, existingKeys: Map[RoleType, KeyPair]) => {
-      existingKeys.filter(_._2.getPublic.id != keyId)
+    keys.computeIfPresent(repoId, (id: RepoId, existingKeys: Map[RoleType, TufKeyPair]) => {
+      existingKeys.filter(_._2.pubkey.id != keyId)
     })
   }
 
   override def fetchTargetKeyPairs(repoId: RepoId): Future[Seq[TufKeyPair]] =  FastFuture.successful {
-    val keyPair = keys.asScala.getOrElse(repoId, throw RoleKeyNotFound).getOrElse(RoleType.TARGETS, throw RoleKeyNotFound)
-
-    Seq(defaultKeyType.crypto.toKeyPair(defaultKeyType.crypto.convertPublic(keyPair.getPublic),
-                                        defaultKeyType.crypto.convertPrivate(keyPair.getPrivate)))
+    Seq(keys.asScala.getOrElse(repoId, throw RoleKeyNotFound).getOrElse(RoleType.TARGETS, throw RoleKeyNotFound))
   }
 
   override def fetchRootRole(repoId: RepoId, version: Int): Future[SignedPayload[RootRole]] =
     fetchRootRole(repoId).filter(_.signed.version == version)
 
   override def fetchKeyPair(repoId: RepoId, keyId: KeyId): Future[TufKeyPair] = Future.fromTry { Try {
-    val keyPair = keys.asScala.getOrElse(repoId, throw KeyPairNotFound).values.find(_.getPublic.id == keyId).getOrElse(throw KeyPairNotFound)
-    val pb = defaultKeyType.crypto.convertPublic(keyPair.getPublic)
-    val prv = defaultKeyType.crypto.convertPrivate(keyPair.getPrivate)
-    defaultKeyType.crypto.toKeyPair(pb, prv)
+    keys.asScala.getOrElse(repoId, throw KeyPairNotFound).values.find(_.pubkey.id == keyId).getOrElse(throw KeyPairNotFound)
```

Comment:
could be `keys.asScala.get(repoId).flatMap(_.values.find(_.pubkey.id == keyId)).getOrElse(throw KeyPairNotFound)`